client/src/utils/CastVoteRecordFiles.ts

```diff
@@ -103,6 +103,35 @@ export default class CastVoteRecordFiles {
     this.allCastVoteRecords = castVoteRecords
   }
 
+  public export() {
+    return JSON.stringify({
+      signatures: JSON.stringify([...this.signatures]),
+      files: JSON.stringify([...this.files]),
+      duplicateFilenames: JSON.stringify([...this.duplicateFilenames]),
+      parseFailedFilenames: JSON.stringify([...this.parseFailedFilenames]),
+      allCastVoteRecords: JSON.stringify(
+        Array.from(this.allCastVoteRecords.entries())
+      ),
+    })
+  }
+
+  public import(stringifiedCVRFiles: string) {
```

Comment:
I think this should be a static method. It completely ignores any content already on the instance, so there's no need for it to be an instance method.

Reply:
done